packages/core/EventManager.ts

```diff
@@ -156,7 +156,7 @@ export default class EventManager {
    *
    * @param event
    */
-  public async update(
+  public async reschedule(
```

Comment:
Since this method in the EventManager is only for rescheduling bookings, I decided to rename it to reflect it's purpose

Reply:
This would need to be updated in every `eventManager.update` call or we're going to have trouble.

Reply:
Agree with @zomars - best for now is to @deprecate the update( function) or update all calls.